geometry_msgs/Pose      brick_pose
geometry_msgs/Pose      feeder_pose
std_msgs/String         brick_type
---
std_msgs/Bool           success
int32                   status
